### ISSUE TITLE
Partial run selection filters out graphs

### ISSUE THREAD

**Issue by pedrob37** [tensorflow/tensorboard#5416]
## Environment information (required)

Please run `diagnose_tensorboard.py` (link below) in the same
environment from which you normally run TensorFlow/TensorBoard, and
paste the output here:

### Diagnostics

<details>
<summary>Diagnostics output</summary>

``````
--- check: autoidentify
INFO: diagnose_tensorboard.py version e43767ef2b648d0d5d57c00f38ccbd38390e38da

--- check: general
INFO: sys.version_info: sys.version_info(major=3, minor=6, micro=9, releaselevel='final', serial=0)
INFO: os.name: posix
INFO: os.uname(): posix.uname_result(sysname='Linux', nodename='dgx1-3', release='4.15.0-91-generic', version='#92-Ubuntu SMP Fri Feb 28 11:09:48 UTC 2020', machine='x86_64')
INFO: sys.getwindowsversion(): N/A

--- check: package_management
INFO: has conda-meta: False
INFO: $VIRTUAL_ENV: None

--- check: installed_packages
INFO: installed: tensorboard==1.15.0+nv
INFO: installed: tensorflow==1.15.2+nv
INFO: installed: tensorflow-estimator==1.15.1

--- check: tensorboard_python_version
INFO: tensorboard.version.VERSION: '1.15.0+nv'

--- check: tensorflow_python_version
2021-11-12 19:31:15.808691: I tensorflow/stream_executor/platform/default/dso_loader.cc:44] Successfully opened dynamic library libcudart.so.10.2
INFO: generated new fontManager
INFO: tensorflow.__version__: '1.15.2'
INFO: tensorflow.__git_version__: 'unknown'

--- check: tensorboard_data_server_version
INFO: no data server installed

--- check: tensorboard_binary_path
INFO: which tensorboard: b'/usr/local/bin/tensorboard\n'

--- check: addrinfos
socket.has_ipv6 = True
socket.AF_UNSPEC = <AddressFamily.AF_UNSPEC: 0>
socket.SOCK_STREAM = <SocketKind.SOCK_STREAM: 1>
socket.AI_ADDRCONFIG = <AddressInfo.AI_ADDRCONFIG: 32>
socket.AI_PASSIVE = <AddressInfo.AI_PASSIVE: 1>
Loopback flags: <AddressInfo.AI_ADDRCONFIG: 32>
Loopback infos: [(<AddressFamily.AF_INET6: 10>, <SocketKind.SOCK_STREAM: 1>, 6, '', ('::1', 0, 0, 0)), (<AddressFamily.AF_INET: 2>, <SocketKind.SOCK_STREAM: 1>, 6, '', ('127.0.0.1', 0))]
Wildcard flags: <AddressInfo.AI_PASSIVE: 1>
Wildcard infos: [(<AddressFamily.AF_INET: 2>, <SocketKind.SOCK_STREAM: 1>, 6, '', ('0.0.0.0', 0)), (<AddressFamily.AF_INET6: 10>, <SocketKind.SOCK_STREAM: 1>, 6, '', ('::', 0, 0, 0))]

--- check: readable_fqdn
INFO: socket.getfqdn(): 'dgx1-3'

--- check: stat_tensorboardinfo
INFO: directory: /tmp/.tensorboard-info
INFO: .tensorboard-info directory does not exist

--- check: source_trees_without_genfiles
INFO: tensorboard_roots (1): ['/usr/local/lib/python3.6/dist-packages']; bad_roots (0): []

--- check: full_pip_freeze
INFO: pip freeze --all:
absl-py==0.9.0
albumentations==1.1.0
astor==0.8.1
attrs==19.3.0
audioread==2.1.8
backcall==0.1.0
bleach==3.1.1
blinker==1.4
captum==0.4.1
cffi==1.14.0
click==8.0.3
cloudpickle==1.3.0
colorama==0.4.4
configparser==5.1.0
cupy==7.2.0
cycler==0.10.0
Cython==0.29.15
dataclasses==0.8
decorator==4.4.2
defusedxml==0.6.0
Deprecated==1.2.13
dm-tree==0.1.6
docopt==0.6.2
efficientnet-pytorch==0.7.1
entrypoints==0.3
fastrlock==0.4
fslpy==3.6.0
future==0.18.2
gast==0.5.2
gitdb==4.0.9
GitPython==3.1.18
google-pasta==0.2.0
graphsurgeon==0.4.1
grpcio==1.27.2
h5py==2.10.0
horovod==0.19.0
humanize==3.12.0
imageio==2.10.3
importlib-metadata==1.5.0
ipykernel==5.1.4
ipython==7.13.0
ipython-genutils==0.2.0
ipywidgets==7.6.5
jedi==0.16.0
Jinja2==2.11.1
joblib==0.14.0
json5==0.9.2
jsonpickle==1.5.2
jsonschema==3.2.0
jupyter==1.0.0
jupyter-client==6.0.0
jupyter-console==6.4.0
jupyter-core==4.6.3
jupyter-tensorboard==0.1.10
jupyterlab==1.2.2
jupyterlab-server==1.0.7
jupyterlab-widgets==1.0.2
jupytext==1.3.4
Keras-Applications==1.0.8
Keras-Preprocessing==1.0.5
kiwisolver==1.1.0
librosa==0.7.1
llvmlite==0.30.0
Markdown==3.2.1
MarkupSafe==1.1.1
matplotlib==3.1.1
mistune==0.8.4
mock==3.0.5
monai==0.7.0
mpi4py==3.0.3
mpmath==1.2.1
munch==2.5.0
nbconvert==5.6.1
nbformat==5.0.4
networkx==2.5.1
nibabel==3.2.1
NiftyNet==0.6.0
ninja==1.10.2.2
nltk==3.2.5
notebook==6.0.2
numba==0.46.0
numpy==1.17.3
nvidia-dali==0.19.0
nvidia-dali-tf-plugin==0.19.0
nvtx-plugins==0.1.7
opencv-python==4.5.4.58
opencv-python-headless==4.5.4.58
opt-einsum==3.2.0
packaging==21.2
pandas==0.25.3
pandocfilters==1.4.2
parso==0.6.2
pexpect==4.7.0
pickleshare==0.7.5
Pillow==8.4.0
pip==21.3.1
portalocker==1.5.2
portpicker==1.3.1
prometheus-client==0.7.1
prompt-toolkit==3.0.3
protobuf==3.11.3
psutil==5.6.5
ptyprocess==0.6.0
py-cpuinfo==8.0.0
py3nvml==0.2.6
pycocotools==2.0.0
pycparser==2.20
pydicom==2.2.2
Pygments==2.5.2
pymongo==3.12.1
pyparsing==2.4.6
pyrsistent==0.15.7
python-dateutil==2.8.1
python-speech-features==0.6
pytz==2019.3
PyWavelets==1.1.1
PyYAML==5.3
pyzmq==19.0.0
qtconsole==5.1.1
QtPy==1.11.2
qudida==0.0.4
resampy==0.2.2
robust-loss-pytorch @ git+https://github.com/jonbarron/robust_loss_pytorch@9831f1db8006105fe7a383312fba0e8bd975e7f6
sacrebleu==1.4.2
sacred==0.8.2
scikit-image==0.17.2
scikit-learn==0.22.2.post1
scipy==1.3.1
seaborn==0.11.2
Send2Trash==1.5.0
sentencepiece==0.1.83
setuptools==58.5.3
SimpleITK==2.1.1
six==1.13.0
sklearn==0.0
smmap==5.0.0
SoundFile==0.10.3.post1
tensorboard==1.15.0+nv
tensorboard-plugin-dlprof==20.3
tensorflow==1.15.2+nv
tensorflow-estimator==1.15.1
tensorflow-probability==0.14.1
tensorrt==7.0.0.11
termcolor==1.1.0
terminado==0.8.3
testpath==0.4.4
tifffile==2020.9.3
torch==1.6.0+cu101
torch-dct==0.1.5
torchio==0.18.62
torchvision==0.7.0+cu101
tornado==6.0.4
tqdm==4.43.0
traitlets==4.3.3
typed-ast==1.4.3
typing==3.7.4.1
typing-extensions==3.10.0.2
uff==0.6.5
wcwidth==0.1.8
webencodings==0.5.1
Werkzeug==1.0.0
wheel==0.34.2
widgetsnbextension==3.5.2
wrapt==1.12.1
xmltodict==0.12.0
zipp==3.1.0

``````

</details>

### Next steps

No action items identified. Please copy ALL of the above output,
including the lines containing only backticks, into your GitHub issue
or comment. Be sure to redact any sensitive information.

For browser-related issues, please additionally specify:

  - Browser type and version (e.g., Chrome 64.0.3282.140): Google Chrome Version 93.0.4577.82 (Official Build) (64-bit)
  - Screenshot, if it’s a visual issue:
 
All runs ticked:
![image](https://user-images.githubusercontent.com/33317198/141523711-8eba5311-b5d1-46a0-b9a4-51c766b5e13c.png)

A single unticked run, note how a lot of shared graphs have disappeared for seemingly no reason:
![image](https://user-images.githubusercontent.com/33317198/141523807-25fce55b-b2e3-459c-a324-12a280e97ab2.png)

## Issue description

I typically display multiple jobs' worth of runs in a single TensorBoard instance. Most of these jobs will share the same graphs, so ideally unticking some runs shouldn't exclude those graphs that display information related to those other runs, but annoyingly this happens consistently and has been a problem for quite some time.

**Comment by UsharaniPagadala** [comment]
@pedrob37 

Could you please refer  the doc for [comparing_different_executions](https://github.com/tensorflow/tensorboard#runs-comparing-different-executions-of-your-model) and also look at similar issues [#36182380](https://stackoverflow.com/questions/36182380/how-do-display-different-runs-in-tensorboard) and  [#1548](https://github.com/tensorflow/tensorflow/issues/1548) and let us know if it helps.Thanks

**Comment by pedrob37** [comment]
@UsharaniPagadala 

Thanks for the links. My problem isn't related to not being able to display multiple runs, it's that entire shared graphs disappear when I start filtering out runs: From my linked images you can see that unselecting the top-most run removes four out of the original nine graphs even though the remaining still-selected runs have data that is contained therein.

**Comment by stephanwlee** [comment]
Hi @pedrob37, it looks like you are using really old version of TensorBoard (tensorboard==1.15.0+nv). I do not think this behavior is reproducible or was ever reported in a newer version of TensorBoard. Would you mind trying a newer version?

**Comment by pedrob37** [comment]
Hi @stephanwlee, thanks for pointing that out, I've since updated my TensorBoard to version 2.2.0, but the problem still seems to persist, in the exact same manner.

**Comment by stephanwlee** [comment]
2.7.0 is the latest TensorBoard. please try the newer version and, if you can, try to use Time Series plugin.

**Comment by UsharaniPagadala** [comment]
@pedrob37 

Have you tried the latest Tensorboard version v2.7?  and let us know if this issue can be closed.Thanks

**Comment by pindinagesh** [comment]
@pedrob37 

We see that you are using old version of Tensorboard, We recommend that you upgrade to Tensorboard version v2.7 and let us know if the issue still persists in newer versions and also refer compatible versions [doc.](https://pypi.org/project/tensorboard/#history) Thanks!

**Comment by pindinagesh** [comment]
@pedrob37 

Closing this issue due to inactivity. Please feel free to reopen if this still exist. Thanks
